### ISSUE TITLE
[CoE Starter Kit - QUESTION] How fix custom connector connection when it breaks / expires

### ISSUE THREAD

**Issue by ibutbul1** [microsoft/coe-starter-kit#2187]
### What is your question?

Hey !

In the process of upgrading the Audit Logs solution there was a need to connect to a custom connector naming "Office 365 Management API" that built by external vendor that gave us services when creating the CoE environment.

When trying to authenticate to that connector there is an error message which there is no application to authenticate to.

![image](https://user-images.githubusercontent.com/101105053/157232862-7b42826a-a827-4d9e-9f45-cbc7a8d52880.png)

That means that we had an Azure application that have been deleted or something in that area ?

Thanks,
Itzhak

**Comment by Jenefer-Monroe** [comment]
Hello. Are you saying that you had someone build a separate custom connector than the one we ship? We do ship a custom connector with that name so I think there may just be confusion. I will assume that is our custom connector for now. 

Note that when you need to recreate the connection (it expires for example), you need to first update the custom connector again. You cannot directly just “fix connection”. I don’t know why this is, but it’s the case. 

See the notes in this GitHub bug for instructions: [CoE Starter Kit - BUG] Audit Logs solution fails after upgrade · Issue #1424 · microsoft/coe-starter-kit (github.com). 

Basically you redo the setup steps from when you originally installed [Setup Custom Connector](https://docs.microsoft.com/power-platform/guidance/coe/setup-auditlog#set-up-the-custom-connector), except you dont need to restart the subscription.

**Comment by ibutbul1** [comment]
Great, now I'm able to connect properly and the flow "[Child] Admin | Sync Logs" can be turned on.

Thanks a lot !

**Comment by Jenefer-Monroe** [comment]
Glad it worked! I'll go ahead and close out, thanks for using CoE
